addons/addon-base-raas/packages/base-raas-services/lib/schema/update-aws-accounts.json

```diff
@@ -31,11 +31,15 @@
       "maxLength": 255,
       "pattern": "^arn:aws:cloudformation:.*$"
     },
+    "cfnStackName": {
+      "type": "string",
+      "maxLength": 2048
+    },
     "roleArn": {
       "type": "string",
       "minLength": 10
     },
-    "xAccEnvMgmtRoleArn": {
```

Comment:
Emmmm we still need the EnvMgmtRole right?

Reply:
we don't, it was actually never saved on the backend. If you look at the develop branch, the `AwsAccount` object doesn't even have an entry for xAccEnvMgmtRoleArn, I think I must've added it later just for this. This role ARN is actually the same for every account, so it shouldn't need to be stored anyway

Reply:
xAccEnvMgmtRoleArn is present in the [create-aws-accounts schema](https://github.com/awslabs/service-workbench-on-aws/blob/mainline/addons/addon-base-raas/packages/base-raas-services/lib/schema/create-aws-accounts.json#L23), and is needed for most of the Service Catalog related calls in the member account. The ARN is different for every account, since it's a role exist in every member account. 

You might have mixed it up with the EnvMgmtRole in the main account, which is used to manage Service Catalog resources in the main account.

Reply:
hm, but it's not present in the [AwsAccount object](https://github.com/awslabs/service-workbench-on-aws/blob/mainline/addons/addon-base-raas-ui/packages/base-raas-ui/src/models/aws-accounts/AwsAccount.js). Is it only kept in the backend?

Reply:
I can add this back in, no problem

Reply:
Yes, it's a miss on the front end side that it's not showed, good catch there! This role is added when the Service Catalog feature is added, and the front end was not updated properly to show this role.